### DISCUSSION TITLE
Event Listener Flow Rubber Duck Chat

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#51] [Rubber Duck]
Ok. I understand the code in chunks, but want to clarify to flow of a Event Listener/Callback. Please let me know if I'm on the right track.
 
<img width="833" alt="Screen Shot 2021-01-24 at 11 23 14 AM" src="https://user-images.githubusercontent.com/68397076/105638108-90789a80-5e36-11eb-9995-2880e5508fbb.png">

**Comment by drteresavasquez** [comment]
Love these notes! This is great and you are on the right track. As we tackle a little more this week it will become more clear but you've got the process down here! 🎉

**Reply by drteresavasquez** [reply]
One other thing to note is you passed the event listener a callback. You can also use an anonymous function in its place.
